src/core/evaluator.js

```diff
@@ -2120,18 +2123,41 @@ class PartialEvaluator {
         }
       }
 
-      var trm = Util.transform(
+      return Util.transform(
         textState.ctm,
         Util.transform(textState.textMatrix, tsm)
       );
-      textContentItem.transform = trm;
+    }
+
+    function ensureTextContentItem() {
+      if (textContentItem.initialized) {
+        return textContentItem;
+      }
+      var font = textState.font;
+      if (!(font.loadedName in seenStyles)) {
+        seenStyles[font.loadedName] = true;
+        textContent.styles[font.loadedName] = {
+          fontFamily: font.fallbackName,
+          ascent: font.ascent,
+          descent: font.descent,
+          vertical: font.vertical,
+        };
+      }
+      textContentItem.fontName = font.loadedName;
```

Comment:
This looks like a bug introduced *accidentally* during a re-basing, since the upstream code changed in the meantime and this won't actually work as intended now; hence you want the following here:
```suggestion
      const font = textState.font,
        loadedName = font.loadedName;
      if (!seenStyles.has(loadedName)) {
        seenStyles.add(loadedName);

        textContent.styles[loadedName] = {
          fontFamily: font.fallbackName,
          ascent: font.ascent,
          descent: font.descent,
          vertical: font.vertical,
        };
      }
      textContentItem.fontName = loadedName;
      
```